aws-cli/plan.ps1

```diff
@@ -0,0 +1,32 @@
+$pkg_name="aws-cli"
+$pkg_origin="core"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=('Apache-2.0')
+$pkg_description="The AWS Command Line Interface (CLI) is a unified tool to \
+  manage your AWS services. With just one tool to download and configure, you \
+  can control multiple AWS services from the command line and automate them \
+  through scripts."
+$pkg_upstream_url="https://aws.amazon.com/cli/"
+$pkg_deps=@(
+  "core/python"
+)
+$pkg_bin_dirs=@("Scripts")
+
+function pkg_version {
+  pip search --disable-pip-version-check awscli | %{ if( $_ -match "^awscli \((.+)\)") { $matches[1]; } }
```

Comment:
I'm surprised this works. I'd expect you would need to run `python -m pip ...`

Reply:
Yeah...me too.

Reply:
it did fail for me locally which likely means your build is catching a different python. I your studio's `/hab/bin` might have a python where pip got installed.

Reply:
okay. well, I'll update to use the `python -m pip` :)